tox.ini

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Moshe Zadka
+# See LICENSE for details.
+[tox]
+envlist = site
+toxworkdir = {toxinidir}/build/tox
+skipsdist = True
+
+[testenv:site]
+deps =
+    pelican
+commands =
+    pelican -t theme/tuxlite_tbs -o {envtmpdir}/output content
+    #python push-gh {envtmpdir}/output git@github.com:pyninsula/pyninsula.github.io.git {posargs}
```

Comment:
It doesn't work right now. I didn't want to gate on debugging it, I push manually.